src/core/compiler.jl

```diff
@@ -288,49 +229,144 @@ function build_model_info(input_expr)
 end
 
 """
-    translate_tilde!(model_info)
+    replace_vi!(model_info)
+
+Replaces @varinfo() expressions to the VarInfo instance.
```

Comment:
Please say what this replaces the `@varinfo()` expressions with.

Also:
```suggestion
Replaces `@varinfo()` expressions in the VarInfo instance.
```